S2S Integration Samples/README.md

```diff
@@ -11,7 +11,7 @@ Before using this sample, you need to onboard to [`Universal Print`](https://aka
 1. Open the solution in Visual Studio and hit 'Restore Nuget Packages'.
 2. Use Visual Studio to build the project.
```

Comment:
Any particular min version of VS that developers need to use?

Reply:
@nilakhan, what's the min version that we use in our Service. We can recommend the same version.

Reply:
lets recommend VS 2019